var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -430,8 +425,10 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("strumpack+shared", when="+strumpack")
     depends_on("suite-sparse", when="+suite-sparse")
     depends_on("superlu-dist", when="+superlu-dist")
-    depends_on("superlu@4.3 +pic", when="+superlu")
-    depends_on("swig", when="@:14 +python")
+    depends_on("superlu@4.3 +pic", when="@:13 +superlu")
+    depends_on("superlu@5.2.2 +pic", when="@14: +superlu")
+    if sys.platform != "win32":
+        depends_on("swig", when="@:14 +python")
```

Comment:
I think python specifically means SWIG ... so it would be strange to enable python but not get the bindings. Does it work to just disable python bindings on windows?

Reply:
Thanks for the insight, I will take a closer look at disabling the python variant on Windows instead.